site/gatsby-site/src/components/discover/createURL.js

```diff
@@ -43,6 +43,8 @@ const convertRangeToQueryString = (range) => {
 const getQueryFromState = ({ state, locale, taxa }) => {
   let query = {};
 
+  if (!state) return;
```

Comment:
This seems like a bad idea -- we don't typecheck or check the return value of this function where it's used:

https://github.com/responsible-ai-collaborative/aiid/blob/1caec141935186372ed9c239951405a4a681b326/site/gatsby-site/src/components/discover/createURL.js#L101

If we want to return a null, we should probably just return null and handle it as an error at the callsite. What do you think?

(apologies for the drive-by comment -- looking at this since we should merge it soon!)